src/d3a/models/strategy/__init__.py

```diff
@@ -161,9 +163,21 @@ def __init__(self):
         super(BaseStrategy, self).__init__()
         self.offers = Offers(self)
         self.enabled = True
+        if ConstSettings.GeneralSettings.EVENT_DISPATCHING_VIA_REDIS:
+            self.redis = RedisMarketCommunicator()
+        self.trade_buffer = None
+        self.offer_buffer = None
```

Comment:
Please indent (didnt think about this back then)

Reply:
Adding it here.